lib/twitter/rest/account_activity.rb

```diff
@@ -118,15 +118,49 @@ def check_subscription(env_name_or_webhook_id)
       # @see https://developer.twitter.com/en/docs/accounts-and-users/subscribe-account-activity/api-reference/aaa-enterprise#delete-account-activity-all-env-name-subscriptions
       # @note Deactivate a subscription, Users events will not be sent to the app
       # @rate_limited Yes
-      # @authentication Requires user context - all consumer and access tokens
+      # @authentication Works with user context and application context
       # @raise [Twitter::Error::Unauthorized] Error raised when supplied user credentials are not valid.
       # @return [nil]
       # @param env_name_or_webhook_id [String] Environment Name or Webhook Id
-      def deactivate_subscription(env_name_or_webhook_id)
+      # @param user_id [String] Id of the user to unsubscribe
+      def deactivate_subscription(env_name_or_webhook_id, user_id: nil)
```

Comment:
```suggestion
      def deactivate_subscription(env_name_or_webhook_id, user_id)
```

`user_id` isn't optional anymore.

Reply:
It's fixed.